spec/CloudCode.spec.js

```diff
@@ -1780,12 +1780,6 @@ describe('afterFind hooks', () => {
   });
 
   it('should validate triggers correctly', () => {
-    expect(() => {
```

Comment:
If we do this we should keep these, but check for a valid response rather than an exception. You can use `.not.ToThrow()` as seen just a bit [below here](https://github.com/addisonElliott/parse-server/blob/36a304b235a93a69b187961cb752983c1783deec/spec/CloudCode.spec.js#L1788).